cmds/init/init.go

```diff
@@ -127,6 +132,14 @@ func main() {
 		}
 	}
 
+	// We can try to prebuild but it's not necessarily and error, since
+	// whoever put this initramfs together might have excluded things.
+	for _, v := range prebuild {
+		if o, err := exec.Command("installcommand", []string{"--onlybuild", v}...).CombinedOutput(); err != nil {
```

Comment:
I'm not even sure this is needed any more, now that we have syscall.Exec in installcommand. I"m gonna drop it.